Cargo.toml

```diff
@@ -25,6 +25,9 @@ ron = "0.8"
 serde = { version = "1", features = ["serde_derive"] }
 shlex = "1"
 tokio = { version = "1", features = ["sync"] }
+# CLI arguments
+lexopt = "0.3.0"
+vergen = "9"
```

Comment:
Not needed. This is already applied in the build.rs: https://github.com/pop-os/cosmic-term/blob/master/build.rs